howto/install-appliance/google-cloud.md

```diff
@@ -0,0 +1,107 @@
+The Anbox Cloud Appliance is not yet available from the Google Marketplace. However, you can install the Anbox Cloud Appliance snap on a Google Cloud instance by using this guide.
+
+The entire deployment process will take 20-30 minutes, depending on the selected hardware and the network conditions.
+
+## Prerequisites
+
+Before starting the procedure,
+* Check the hardware requirements for the Anbox Cloud Appliance [here](https://discourse.ubuntu.com/t/17734).
+* Make sure that you have a Google Cloud account and a project on Google Cloud to create the virtual machine.
+* If you wish to use your own Ubuntu Pro subscription, ensure you have the Ubuntu Pro token for your Ubuntu Pro subscription. If you wish to use the Ubuntu Pro subscription offered by Google Cloud along with the virtual machine, skip this prerequisite.
```

Comment:
For the use of  `your own Ubuntu Pro subscription`, can we link to https://ubuntu.com/pro?